### ISSUE TITLE
Create RPM Package

### ISSUE THREAD

**Issue by turicas** [jdunck/python-unicodecsv#66]
This work is already being made by some Fedora contributors and (I expect) will be available soon in the official repositories.
You may want to watch the process: https://bugzilla.redhat.com/show_bug.cgi?id=1266273

> Note: I'm watching becase [a library I develop](https://github.com/turicas/rows) depends on unicodecsv and I want to have it packaged to Fedora.

**Comment by apevec** [comment]
Thanks for watching, python-unicodecsv-0.14.1-1.fc24 is now available in Fedora Rawhide.
BTW I would like to drop python3-* which doesn't seem to be necessary and I had to patch python-cliff 1.15 to pass py3 tests, see https://review.openstack.org/226966

**Comment by turicas** [comment]
I think the Python 3 package is not necessary (I'm not contrary to removing it) but it could be very useful since we (as `unicodecsv` users) don't need to check Python version and use `unicodecsv` or `csv` (we can just use `unicodecsv`).

@jdunck, what do you think?

**Comment by apevec** [comment]
See https://github.com/jdunck/python-unicodecsv/issues/65 where it was mentioned:

> You're using unicodecsv as a drop-in replacement for csv. The problem is that it's only a reasonable drop-in replacement for Python 2 csv. If it attempted to be a drop-in replacement for Python 3's as well, it would be a different API

**Comment by turicas** [comment]
@apevec, you're right. :)

**Comment by ryanhiebert** [comment]
Yeah, at least with 0.14.1, that's certainly the case. I have #67 open that compromises the mental model a bit so that it _can_ be a drop-in replacement. I'd love feedback on whether that magical approach is really a good idea or not.

**Comment by apevec** [comment]
AFAICT conclusion in https://github.com/jdunck/python-unicodecsv/pull/67 is to keep advertising py3 compatibility but document clearlz that it is not a drop-in replacement for py3 csv module.
As such, Fedora RPM will keep python3-unicodecsv subpackage with modified summary:
-Summary:        Drop-in replacement for csv module which supports unicode strings
+Summary:         Python 2/3 csv shim, not a replacement for py3 csv module

Better wording suggestions welcome!

**Comment by ryanhiebert** [comment]
```diff
-Summary: Drop-in replacement for csv module which supports unicode strings
+Summary: Drop-in replacement for Python 2.7's csv module which supports unicode strings
```

It's the same distinction as the README gives.

If there's a longer description that a one-line summary, a note that it's _not_ a replacement for Python 3's csv module is appropriate. But you can't put everything in a single summary line.
